source/security/authentication.txt

```diff
@@ -0,0 +1,545 @@
+.. _pymongo-auth:
+
+Authentication Mechanisms
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: authorize, boto, ecs, aws
+
+Overview
+--------
+
+This guide describes the mechanisms you can use in {+driver-short+} to authenticate
+users.
+
+.. important:: Percent-Escaping
+
+   You must percent-escape a username and password before including them
```

Comment:
This page mentions percent-escaping throughout, but I don't actually see any examples of what that means. I think an example showing it in this admonition would be helpful

Reply:
Added a wiki link and an example. Also updated to "percent-encoding", apparently the correct term